src/editors/VideoSelectorPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import PropTypes from 'prop-types';
+import ErrorBoundary from './sharedComponents/ErrorBoundary';
+import { VideoSelector } from './VideoSelector';
+import store from './data/store';
+
+const VideoSelectorPage = ({
+  courseId,
+  lmsEndpointUrl,
+  studioEndpointUrl,
```

Comment:
I think it might be better to pass props like these via Context instead of drilling them from one component to the next.

Reply:
Yes you are right, can we change this into another PR? The different components of this repo use this approach and I have followed it to keep everything on the same line.